command-builder/ng-add/index.ts

```diff
@@ -0,0 +1,126 @@
+import { Rule, SchematicContext, SchematicsException, Tree, chain } from '@angular-devkit/schematics';
+import { experimental, JsonParseMode, parseJson } from '@angular-devkit/core';
+import {
+    addPackageJsonDependency,
+    NodeDependency,
+    NodeDependencyType
+} from 'schematics-utilities';
+
+function addPackageJsonDependencies(): Rule {
+    return (host: Tree, context: SchematicContext) => {
+        const dependencies: NodeDependency[] = [
+            { type: NodeDependencyType.Default, version: '~1.9.0', name: 'netlify-builder/deploy' }
```

Comment:
Here you can change the version to 1.0.3, will update the same on release.

Reply:
can you please edit the version and accept the PL.

Reply:
cool